raiden/api/python.py

```diff
@@ -65,6 +65,32 @@ def get_channel(self, channel_address):
 
         raise ValueError("Channel not found")
 
+    def manager_address_if_token_registered(self, token_address):
+        """
+        If the token is registered then, return the channel manager address.
+        Returns None otherwise.
+        """
+        try:
+            manager = self.raiden.chain.manager_by_token(token_address)
```

Comment:
For what I've seen, `manager_by_token` doesn't raise an exception. If the token isn't registered, `proxy.channelManagerByToken.call(token_address)` returns `''`. Maybe an `assert` or `if / else` instead of `try/except` would do the trick.